specification.markdown

```diff
@@ -427,17 +417,30 @@ The id attribute is REQUIRED.
 
 Within an EBML Schema, the XPath of `@minOccurs` attribute is `/EBMLSchema/element/@minOccurs`.
 
-An integer expressing the minimum permitted number of occurrences of this EBML Element within its Parent Element. The minOccurs value MUST be equal to the EBMLMinOccurrence value of the path.
+The minOccurs is an integer representing the minimum permitted number of occurrences of this EBML Element within its Parent Element.
+
+Each instance of the Parent Element MUST contain at least this many instances of this EBML Element.
+If the EBML Element has an empty EBMLParentPath then minOccurs refers to constraints on the occurrence of the EBML Element within the EBML Document.
+EBML Elements with minOccurs set to "1" that also have a default value (see (#default)) declared are not REQUIRED to be stored but are REQUIRED to be interpreted, see (#note-on-the-use-of-default-attributes-to-define-mandatory-ebml-elements).
+
+An EBML Element defined with a minOccurs value greater than zero is called a Mandatory EBML Element.
 
 The minOccurs attribute is OPTIONAL. If the minOccurs attribute is not present then that EBML Element has a minOccurs value of 0.
 
+The semantic meaning of minOccurs within an EBML Schema is analogous to the meaning of minOccurs within an XML Schema.
+
 #### maxOccurs
 
 Within an EBML Schema, the XPath of `@maxOccurs` attribute is `/EBMLSchema/element/@maxOccurs`.
 
-An integer expressing the maximum permitted number of occurrences of this EBML Element within its Parent Element. The maxOccurs value MUST be equal to the EBMLMaxOccurrence value of the path.
+The maxOccurs is an integer representing the maximum permitted number of occurrences of this EBML Element within its Parent Element.
+
+Each instance of the Parent Element MUST contain at most this many instances of this EBML Element.
+If the EBML Element has an empty EBMLParentPath then maxOccurs refers to constraints on the occurrence of the EBML Element within the EBML Document.
+
+The maxOccurs attribute is OPTIONAL. If the maxOccurs attribute is not present then there is no upper bound for the permitted number of occurrences of this EBML Element within its Parent Element or within the EBML Document depending on whether the EBMLParentPath of the EBML Element is empty or not.
 
-The maxOccurs attribute is OPTIONAL. If the maxOccurs attribute is not present then that EBML Element has no maximum occurrence, similar to unbounded in the XML world.
+The semantic meaning of maxOccurs within an EBML Schema is analogous to the meaning of unbounded within an XML Schema.
```

Comment:
This doesn't match line 430 well. I think you mean to say that ebml:minOccurs is like xml:minOccurs but then say ebml:maxOccurs is like xml:unbounded. But it is like xml:unbounded. I suppose an unstated ebml:maxOccurs is like an xml:maxOccurs that is set to "unbounded".